java/compat/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
We already have one of these. Can we keep them in some shared location and have each directories just invoke the shared ones?